libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx

```diff
@@ -101,6 +126,7 @@ export const EmbedInsightDialogBaseExamples: React.VFC = () => {
                         action("onOptionChange");
                     }}
                     openSaveInsightDialog={action("openSaveInsightDialog")}
+                    showWebComponentsTab={true}
```

Comment:
Please move this prop to line 120, as props which are non-function should be next to each other, for the sake of consistency.